test-data/src/test/resources/org/openmrs/module/fhir2/providers/TestTask_create.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+-->
+<Task xmlns="http://hl7.org/fhir">
+    <extension url="https://fhir.openmrs.org/ext/task">
+      
+        </extension>
```

Comment:
So, ideally, this would be identical to the Task created in the JSON file. The XML form of that JSON file would look something like this:

```xml
<Task xmlns="http://hl7.org/fhir">
	<id value="d899333c-5bd4-45cc-b1e7-2f9542dbcbf6"/>
	<status value="accepted"/>
	<intent value="order"/>
	<authoredOn value="2016-10-31T08:25:05+10:00"/>
</Task>
```